client/client.go

```diff
@@ -62,6 +62,7 @@ type Client struct {
 	conn  *imap.Conn
 	isTLS bool
 
+	replies   chan string
```

Comment:
Can this take a `[]byte` instead of a `string`? This would make it more obvious that it directly writes to the socket,

Reply:
ok